docs/sites.yml

```diff
@@ -10075,6 +10075,16 @@
     - Library
   built_by: Gilbish Kosma
   built_by_url: https://www.gil20.me/
+  - title: Monolit
```

Comment:
```suggestion
- title: Monolit
```

This should probably fix it

Reply:
Hi, I've just committed a fix as you suggested.